content/sql/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'SQRT()'
+Description: 'Returns the square root of a given number.'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'MySQL'
+  - 'PostgreSQL'
+  - 'SQLite'
+  - 'SQL Server'
+CatalogContent: 
+  - 'learn-sql'
+  - 'path/analyze-data-with-sql'
+---
+
+**SQRT()** is a function in SQL that returns the square root of a number input. The square root of any given number is a number that multiplied by itself will result in the number. 
+
+## Syntax
+
+```sql
+SELECT SQRT(number);
+```
+
```

Comment:
Like to see a sentence after the syntax code block explaining what `number` is.

Reply:
@DaMull Did you push these changes, I don't see them in this PR.

Reply:
@SSwiniarski 

I ran into some issues that I had to google and figure out. I pushed the commits now. Sorry if some are a little redundant from me trying to troubleshoot. Thanks for yall's patience!